docs/1.getting-started/6.data-fetching.md

```diff
@@ -149,7 +149,11 @@ const { pending, data: posts } = useLazyFetch('/api/posts')
 
 ### Client-only fetching
 
-By default, data fetching composables will perform their asynchronous function on both client and server environments. Set the `server` option to `false` to only perform the call on the client-side. Combined with the `lazy` option, this can be useful for data that are not needed on the first render (for example, non-SEO sensitive data).
+By default, data fetching composables will perform their asynchronous function on both client and server environments. Set the `server` option to `false` to only perform the call on the client-side. On initial load, the data will not be fetched before hydration is complete so you have to handle a pending state.
+
+On the client-side navigation the data will be loaded before.
+
+Combined with the `lazy` option, this can be useful for data that are not needed on the first render (for example, non-SEO sensitive data).
```

Comment:
```suggestion
Combined with the `lazy` option, this can be useful for data that is not needed on the first render (for example, non-SEO sensitive data).
```